### ISSUE TITLE
why remove export statement

### ISSUE THREAD

**Issue by lovetingyuan** [coderaiser/putout#134]
input: a.ts
```typescript
export type GetA = typeof getA

export const getA = () => {
  return false
}

```
after run `npx putout --fix a.ts`
output: 
```typescript
const getA = () => false;
export type GetA = typeof getA;

```

function getA is a exported value, it is used by other modules, why remove `export`?

**Comment by coderaiser** [comment]
Fixed with 39a973e 🎉, please re-install 🐊**Putout**, and check again, is it works for you?

**Comment by lovetingyuan** [comment]
yes, the problem solved, I will close it, thank you!
